app/views/pages/setup.html.erb

```diff
@@ -54,7 +54,13 @@
         <li>Visit <code>https://github.com/$ORGANIZATION/$REPO/settings/hooks/new</code>.</li>
         <li>Enter <code><%= ENV["WEBHOOK_URL"] %></code> for the Payload URL.</li>
         <li>Enter <code><%= ENV["GITHUB_SECRET_KEY"] %></code> for the secret.</li>
-        <li>Select <code>Send me everything</code>, and submit.</li>
+        <li>Select following event types under "Let me select individual events":
```

Comment:
I think adding logging for debugging is sufficient. I still stand pretty strong against selecting events.